### PR TITLE
Refactor governance content

### PR DESCRIPTION
**Pull request by jeffabailey**
The goal of this PR is to move content centered around InnerSource governance to an appropriate location.

See this PR for reference: 	https://github.com/InnerSourceCommons/managing-innersource-projects/pull/40

### CODE REVIEW

File: introduction/framework.md [hunk 1]
```diff
@@ -32,40 +32,9 @@ Let's introduce the *InnerSource framework*.
...
--  people roles and responsibilities definitions
-
--  community support mechanisms 
-
--  decision making process policies description
-
--  contribution process policies description
-
--  monitoring policies and mechanisms description
+Pick the right [governance standards and model](/governance/governance.md) for your organization.
```
Comment (jeffabailey) [inline introduction/framework.md:37]: @rrrutledge @spier 

Given this context, I question whether or not linking to this file is the right approach.

https://github.com/InnerSourceCommons/managing-innersource-projects/pull/40#discussion_r1375516320

image

We need to fundamentally decide if a hard copy of this book is a project goal. If it is, then we need to refer to the content with footnotes as suggested.

Comment (rrrutledge) [reply introduction/framework.md:37]: I think that online book is what we're going for.

**Review by rrrutledge — APPROVED** [review]

File: introduction/framework.md (hunk 1 above)
Comment (spier) [reply introduction/framework.md:37]: I would also say that an online book (effectively just a website) is our current target format.
gitbook provides us this format, as well as the possibility to do a PDF export, where links will likely be clickable as well.

**Comment by JustinGOSSES** [comment]
@jeffabailey and @fioddor Is there any specific feedback one of you still needs to integrate here or can anyone approve and merge this now?

**Comment by jeffabailey** [comment]
> @jeffabailey and @fioddor Is there any specific feedback one of you still needs to integrate here or can anyone approve and merge this now?

We have one approval.

If we can get one more, then I'll merge the change.

**Review by dellagustin-sap — APPROVED** [review]
Looks good to me, the content behind the link covers the content that was removed.

**Review by JustinGOSSES — APPROVED** [review]

**Pull request merged**